tests/unit/README.md

```diff
@@ -0,0 +1,59 @@
+# OpenThread Unit Tests
+
+This page shows how to build and run OpenThread unit tests. It will be helpful for developers to debug failed unit test cases if they got one in CI or to add some new test cases.
+
+## Build Simulation
+
+The unit tests cannot be built solely without building the whole project. So first build OpenThread on simulation platform:
+
+```
+# Go to the root directory of OpenThread
+./script/cmake-build simulation
+```
+
+Now all the unit tests should have been built.
+
+## List all tests
+
+To see what tests are there in OpenThread:
+
+```
+# Make sure you are at the simulation build directory (build/simulation)
+ctest -N
```

Comment:
As mentioned earlier, add the terminal prompt for this command and any others below 

`$ ctest -N`